src/app/header/header.component.html

```diff
@@ -1,18 +1,20 @@
 <header>
-  <nav class="navbar navbar-dark bg-primary navbar-expand-md">
-    <div [ngClass]="{'clearfix': !(isNavBarCollapsed | async)}">
-      <a class="navbar-brand" routerLink="/home">{{ 'title' | translate }}</a>
+    <div class="container">
+        <a class="navbar-brand my-2" routerLink="/home">
+            <img src="assets/images/dspace-logo.svg"/>
+        </a>
+
+        <nav class="navbar navbar-light navbar-expand-md float-right px-0">
+            <a href="#" class="px-1"><i class="fas fa-search fa-lg fa-fw"></i></a>
```

Comment:
`<i>` tag needs a "title" for accessibility.

Also, I notice that when I test this PR, clicking this search icon loses my login information.  I'm assuming though that this is likely because it doesn't yet link to anywhere.